### ISSUE TITLE
Failure of WCAG SC 1.4.1 Use of Color in "Results of questionnaire" heading

### ISSUE THREAD

**Issue by matatk** [w3c/wbs-design#141]
When viewing the results of a survey, there is an `<h1>` that contains the text "Results of Questionnaire [survey name]". The survey name text is actually a link to the survey itself (to fill it in).

The link is differentiated from the adjacent text by way of color alone, which fails WCAG.

To resolve this, you need to provide some visual indication _other than color_ for the link text. The best way to do this is to underline the link. Technically you could do other stuff, like make the link text bold, but many people will struggle to see the difference. Underlining the link is the standard, and a very clear, way of indicating that a link is a link.

![The heading, with text "Results of Questionnaire Renaming Personalization Modules", and having some words in black and the text of the link in blue](https://user-images.githubusercontent.com/1125522/152031961-5ea4d899-51c5-4392-92e5-1145ae403c18.png)
This comes from https://www.w3.org/2002/09/wbs/101569/name2022/results

**Comment by lcarcone** [comment]
Thanks for the report.
I changed the style to underline the link in the <h1>
Regards

**Comment by lcarcone** [comment]
Thanks for the report.
I changed the style to underline the link in the h1.
Regards
